xml/ha_storage_protection.xml

```diff
@@ -781,18 +780,44 @@ Timeout (msgwait)  : 10
        would be appropriate if the <literal>msgwait</literal> timeout value for
        SBD was set to <literal>30</literal> seconds.</para>
      </callout>
-     <callout arearefs="co.ha.sbd.st.rsc">
-      <para>
-       The &stonith; resource to use for SBD.</para>
-     </callout>
-     <callout arearefs="co.ha.sbd.pm_delay_max">
+   </calloutlist>
+  </step>
+  <step>
+   <para>For a two-node cluster, decide if you want predictable or random delays. For
+   other cluster setups you do not need to set this parameter.</para>
+   <variablelist>
+    <varlistentry>
+     <term>Predictable Static Delays</term>
+     <listitem>
+      <para>This parameter enables a static delay before executing &stonith; actions.
+      It ensures that the nodes do not fence each other if separate fencing resources
+      and different delay values are being used. The targeted node will
```

Comment:
How about something like:

By configuring separate fencing resources with different delay values, it prevents the nodes from getting reset at the same time in case of a split-brain scenario in a two-node cluster.

Reply:
Technically it is correct. :slightly_smiling_face: For my taste, I feel the sentences should be splitted. Furthermore, what is "it" in your sentence?

I've tried to solve that by rearranging it:

```
To prevent the nodes form getting reset at the same time, configure separate fencing resources with
different delay values. This [approach] helps to avoid a split-brain scenario in a two-node cluster.
```

Sure, it emphasize the sentence a bit differently. What do you think?

Reply:
We cannot prevent split-brain from happening :-) What we are doing here is to prevent "double-fencing" from happening under split-brain of two-node cluster. So probably something like this:

"To prevent the nodes from getting reset at the same time under split-brain of a 2-node cluster, configure separate fencing resources with different delay values."

?

Reply:
Haha, right. Thanks, much better! I use your sentence.